shell/platform/windows/testing/view_modifier.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_TESTING_VIEW_MODIFIER_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_TESTING_VIEW_MODIFIER_H_
+
+#include "flutter/shell/platform/windows/flutter_windows_view.h"
+
+#include "flutter/fml/macros.h"
+#include "flutter/shell/platform/windows/testing/egl/mock_window_surface.h"
+
+namespace flutter {
+
+// A test utility class providing the ability to access and alter various
+// private fields in a |FlutterWindowsView| instance.
+class ViewModifier {
```

Comment:
TODO: CI license check isn't failing for this?

Reply:
Its in a "testing" folder. The license checker skips those.

Reply:
Ah TIL, thanks for the info!